cachito/workers/tasks/gomod.py

```diff
@@ -54,6 +55,9 @@ def fetch_gomod_source(request_id, dep_replacements=None, package_configs=None):
     :param list package_configs: the list of optional package configurations submitted by the user
     :raises CachitoError: if the dependencies could not be retrieved
     """
+    version_output = run_cmd(["go", "version"], {})
```

Comment:
I would be really glad if we add supported versions to documentation as well. Some users ask about it first before submitting a request. And we had a few questions about it before. We need to answer them every time and it could be better just to point to the Cachito docs. 

 Please, tell me if you need help with it.

Reply:
Which kind of docs do you mean? README, some document in `docs/` directory or some Confluence page?

Reply:
README will be the best. There is a part for every package manager

Reply:
Updated.